@kiva/kv-components/utils/expander.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-param-reassign */
+
+function setInitialStyle(el, { property, delay, easing }) {
```

Comment:
Ideally these functions get comments, but I see these are copied from the `ui` repo, so not a huge priority, since we didn't have comments there either, and our utils in this repo don't always have comments.

Reply:
In other words, don't worry about comments in this PR, but I thought I'd at least mention that comments are ideal 😄.